### PR TITLE
Styling updates

### PR DESCRIPTION
**Pull request by Shinsina**
Screen Shot 2021-07-28 at 11 03 52 AM
Screen Shot 2021-07-28 at 11 41 43 AM

BEFORE:

Screen Shot 2021-07-29 at 8 20 50 AM
Screen Shot 2021-07-29 at 8 20 58 AM

AFTER:

Screen Shot 2021-07-29 at 8 17 51 AM
Screen Shot 2021-07-29 at 8 18 20 AM

Before finalized line height:
Screen Shot 2021-07-29 at 9 56 41 AM

After finalized line height:
Screen Shot 2021-07-29 at 9 56 29 AM

### CODE REVIEW

**Comment by Shinsina** [comment]
[LOW URGENCY: Increase font size by 1 pt. in all header text](https://trello.com/c/cs3WiA5x/78-medium-urgency-increase-font-size-by-1-pt-in-all-header-text)

**Comment by Shinsina** [comment]
[LOW URGENCY: Increase line height on department headers below hero spot.](https://trello.com/c/0zsiCTNg/79-low-urgency-increase-line-height-on-department-headers-below-hero-spot)

**Comment by Shinsina** [comment]
[LOW URGENCY: Fix font bolding issue on Media Gallery.](https://trello.com/c/Dsb983HE/80-low-urgency-fix-font-bolding-issue-on-media-gallery)

**Comment by Shinsina** [comment]
Screen Shot 2021-07-28 at 12 59 53 PM

File: packages/global/scss/index.scss [hunk 1]
```diff
@@ -709,6 +713,11 @@ body {
 .section-card-block {
   &__header {
     @include theme-card-header();
+    font-size: 32px;
+    font-weight: 800;
+    line-height: .5;
```
Comment (zarathustra323) [inline packages/global/scss/index.scss:718]: I would ask Ashton the actual line height value to use instead of "decrease by 1pt"

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

**Comment by zarathustra323** [comment]
For clarity, can you provide before and after screen shots and update your first PR comment to show them (as opposed to adding new comments).

File: packages/global/scss/index.scss [hunk 2]
```diff
@@ -709,6 +713,11 @@ body {
 .section-card-block {
   &__header {
     @include theme-card-header();
+    font-size: 32px;
+    font-weight: 800;
```
Comment (B77Mills) [inline packages/global/scss/index.scss:717]: 800 is not a supported font weight for the font family being used.  [Oswald](https://fonts.google.com/specimen/Oswald) bold is 700 which is being inherited from the @include theme-card-header.  I would just remove the font-weight override.

**Comment by Shinsina** [comment]
> For clarity, can you provide before and after screen shots and update your first PR comment to show them (as opposed to adding new comments).

Comment updated

**Review by zarathustra323 — APPROVED** [review]

**Pull request merged**